### ISSUE TITLE
Response handling broken in some systems "Warning: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed"

### ISSUE THREAD

**Issue by ralphcallaway** [joeferraro/MavensMate-SublimeText#572]
python warning is causing response handling to break in some cases, in particular linux builds

@brettbatie has a pull request #489 that drops tracking standard error that may fix this, but don't know if standard error is important in any other cases

known issue for python https://bugs.launchpad.net/ubuntu/+source/python-defaults/+bug/962639

looks like it's been around with 2.7 for two years now, waiting on a fix may not be realistic

example mm console output
```
Operation: Refreshing Selected Metadata
Timestamp: Mon, 04 May 2015 08:23:23
   Result: [OPERATION FAILED]: Whoops, unable to parse the response. Please enable logging (http://mavensmate.com/Plugins/Sublime_Text/Plugin_Logging) and post relevant log(s) to a new issue at https://github.com/joeferraro/MavensMate-SublimeText
[RESPONSE FROM MAVENSMATE]: /usr/lib/python2.7/dist-packages/gobject/constants.py:24: Warning: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed

  import gobject._gobject

{

    "body": "Refresh Completed Successfully", 

    "body_type": "text", 

    "success": true, 

    "time": "6.71831488609314"

}
```

**Comment by ralphcallaway** [comment]
@joeferraro does the mm core api utilized standard error for communication back to the ST3 plugin? 

@brettbatie has a fix for this, but it involves ignoring stderror from mm commands, and was curious if that was something you definitely aren't using, or whether you are definitely using it in some cases
